src/frontend/komponenter/Oppgavebenk/Oppgavebenk.tsx

```diff
@@ -6,7 +6,7 @@ import OppgaveHeader from './OppgaveHeader';
 import OppgaveList from './OppgaveList';
 import { useAmplitude } from '../../utils/amplitude';
 
-const StyledDiv = styled.div`
+const Container = styled.article`
```

Comment:
Blir det riktig å bruke article her? 🤔

Reply:
Vi lander på at dette blir greit, med sections for listen og headeren